### PR TITLE
Milestone Three: Style Components

### PR DESCRIPTION
**Pull request by phylischumba**
## Add styles to the project
- Added style to the components
- Added props to the button component
- Fix linter errors

### CODE REVIEW

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @Phylis05 you did well with this implementation, it looks really nice but you want the App component to  have flex display 
see this

**Please make changes and Request for another review** 

**RAPHAEL NORIODE** :mask: STAY_SAFE

**Comment by phylischumba** [comment]
Hey @Oghenebrume50,

Thanks for the review. I have implemented the change you just requested

File: src/index.css [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
-  margin: auto;
-}
-
-.result {
-  color: green;
+.app {
+  width: 700px;
+  display: flex;
+  flex-direction: column;
 }
```
Comment (shloch) [inline src/index.css]: you can center the app, for symmetry
```
margin: auto;
```

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @Phylis05 

Hello Great job with incorporating the `flex` .........
SOrry i will have to come back to some errors which is very important

- I have console errors, please debug this
calc

## [optional]
- it's about the comment below

Happy coding

**Comment by phylischumba** [comment]
Hey @shloch,
Thanks for picking this in the review. I have implemented the change as requested

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :+1: 
Hi @Phylis05 :hand: 
Great job on this milestone. You may merge the PR now.
Good luck with the next one.

Happy Coding :computer: 
**Sergio Torres**

**Comment by phylischumba** [comment]
Thanks @Torres-ssf  for the approval

**Pull request merged**